src/d3a/models/strategy/external_strategies/pv.py

```diff
@@ -316,6 +315,8 @@ def _update_offer_aggregator(self, arguments):
                     offer_arguments['energy'] = offer.energy
                     offer_arguments['price'] = \
                         (offer_arguments['price'] / offer_arguments['energy']) * offer.energy
+                    offer_arguments["seller"] = offer.seller
```

Comment:
Please also add this in the external Storage

Reply:
@hannesdiedrich Done